crates/wasmprinter/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmprinter"
-version = "0.2.23"
+version = "0.3.0"
```

Comment:
Since this one didn't have any API-breaking changes we leave wasmprinter on the 0.2 track, so this can just bump it to 0.2.24